README.md

```diff
@@ -1,38 +1,19 @@
-# [DEPRECATED] FakerBot
-
-[![codebeat badge](https://codebeat.co/badges/ef4839f2-3946-4435-8c0e-7025de5fac22)](https://codebeat.co/projects/github-com-akabiru-fakerbot-master) [![Inline docs](http://inch-ci.org/github/akabiru/fakerbot.svg?branch=master)](http://inch-ci.org/github/akabiru/fakerbot) [![Build Status](https://travis-ci.com/akabiru/fakerbot.svg?branch=master)](https://travis-ci.com/akabiru/fakerbot) [![Coverage Status](https://coveralls.io/repos/github/akabiru/fakerbot/badge.svg?branch=master)](https://coveralls.io/github/akabiru/fakerbot?branch=master) [![Gem Version](https://badge.fury.io/rb/fakerbot.svg)](https://badge.fury.io/rb/fakerbot) [![Open Source Helpers](https://www.codetriage.com/akabiru/fakerbot/badges/users.svg)](https://www.codetriage.com/akabiru/fakerbot) [![Maintainability](https://api.codeclimate.com/v1/badges/58e23da2916f5007b5d3/maintainability)](https://codeclimate.com/github/akabiru/fakerbot/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/58e23da2916f5007b5d3/test_coverage)](https://codeclimate.com/github/akabiru/fakerbot/test_coverage)
+# faker-cli
```

Comment:
Good stuff. I think it would be good to get the CI & Code coverage badges back, this time tied to the Org. Will add an issue!

Reply:
absolutely